src/run/pindle.py

```diff
@@ -38,7 +38,7 @@ def approach(self, start_loc: Location) -> Union[bool, Location]:
             return False
         wait(0.5, 0.6)
         found_loading_screen_func = lambda: self._ui_manager.wait_for_loading_screen(2.0)
-        if not self._char.select_by_template(["A5_RED_PORTAL", "A5_RED_PORTAL_TEXT"], found_loading_screen_func):
+        if not self._char.select_by_template(["A5_RED_PORTAL", "A5_RED_PORTAL_TEXT"], found_loading_screen_func, telekinesis=True):
```

Comment:
Specifically this we can not do, because using telekinesis will make the sorc end up on a different starting position than when entering the portal by foot. Then the static pathing is all messed up.

Reply:
I tested in act5 and works flawless